portfolio/src/main/java/com/google/sps/data/PortfolioComments.java

```diff
@@ -0,0 +1,43 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/** Class containing server statistics. */
+public final class PortfolioComments {
+
+	private final List<String> masterCommentList = new ArrayList<>();
+
```

Comment:
indentation. 

remove extra empty line. Empty line is more like a paragraph and only add when it helps readability. No need to have it between a list declaration and function used it below?